src/include/migraphx/pass.hpp

```diff
@@ -24,20 +26,45 @@ struct pass
     /// A unique name used to identify the pass
     std::string name() const;
     /// Run the pass on the module
+    void apply(module_pass_manager& mpm) const;
     void apply(module& m) const;
```

Comment:
Do we need this method that applies on Module directly ?

Reply:
Yes as all of our passes are written using this method.

Reply:
Then, Should all of them now use module_pass_manager instead 
?

Reply:
I dont see a reason to change it.